template/eip.tpl

```diff
@@ -6,4 +6,14 @@ export PATH=~/.local/bin:$PATH
 
 pip install aws-ec2-assign-elastic-ip
 export AWS_DEFAULT_REGION=$(curl -s http://169.254.169.254/latest/dynamic/instance-identity/document | grep region | awk -F\" '{print $4}')
-/bin/aws-ec2-assign-elastic-ip --valid-ips ${eip}
\ No newline at end of file
+/bin/aws-ec2-assign-elastic-ip --valid-ips ${eip}
+
+# The AWS CLI is a requirement of the GitLab backup and restore scripts
```

Comment:
Remove or change this comment to avoid confusion (we're not running backup/restore scripts on the gitlab-runner).

Reply:
Thanks for catching this. I've updated the comment to simply state we are installing aws cli.